go.mod

```diff
@@ -4,58 +4,33 @@ go 1.13
 
 require (
```

Comment:
Update also the go version to 1.14

Reply:
I thought about having the CI running at 1.14 first, and then updating this. I'm not sure our downstream builds are able to run 1.14 yet.

cc @kevinearls and @objectiser

Reply:
Correct - still on 1.13 - however we should be able to move to 1.14 here and pick up any issues when the internal CI builds run.